.stickler.yml

```diff
@@ -0,0 +1,4 @@
+linters:
```

Comment:
Hey team, some of your stickler issues, other than the indentation may actually be due to the fact that your `.stickler.yml` file does not ignore certain files.

```yml
  
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```